### PR TITLE
completed seeding the database. was able to get the data to render sp…

### PR DESCRIPTION
**Pull request by ikaur1718**
…completed seeding the database. I was able to get the data to render specs and shipping info.

### CODE REVIEW

**Comment by ikaur1718** [comment]
@benmangold Please review, I rendered Specs, shipping and included components with the dynamic data coming from the database.

File: database-mongo/index.js [hunk 1]
```diff
@@ -1,31 +1,68 @@
...
+    images: info.images,
+    shippingDate: info.shippingDate,
+    details: info.details
+  }).save();
+  callback(null);
+};
 
-// module.exports.selectAll = selectAll;
\ No newline at end of file
+var seedDatabase = () => {
```
Comment (benmangold) [inline database-mongo/index.js:50]: nice!  i like the way you did this

File: server/index.js [hunk 2]
```diff
@@ -2,14 +2,37 @@ var express = require('express');
...
-  console.log('listening on port 3000!');
+app.get('/buy/:prod_name/overview', (req, res) => {
+  console.log('In get ', req.params.prod_name);
+  db.getOverviewData(req.params.prod_name)
+  .then((data) => {
+    console.log(data);
+    res.status(201).send(data);
+  })
+  .catch((err) => {
+    console.error(err);
```
Comment (benmangold) [inline server/index.js:20]: i think this will leave the client hanging upon error, instead of returning 500

File: react-client/src/index.jsx [hunk 3]
```diff
@@ -36,29 +37,70 @@ class App extends React.Component {
...
+            specs: response.data[0].specs,
+            included: response.data[0].included,
+            boxContent: response.data[0].boxContent,
+            descriptionHeader: response.data[0].descriptionHeader,
+            description: response.data[0].description,
+            imageHeader: response.data[0].imageHeader,
+            images: response.data[0].images,
+            shippingDate: response.data[0].shippingDate,
+            details: response.data[0].details,
+            prod_name: response.data[0].prod_name
```
Comment (benmangold) [inline react-client/src/index.jsx:82]: dang, so much data

**Review by benmangold — APPROVED** [review]
Looks great!  In general there are some comments and console logs floating around which should eventually get cleared out, but all the functionality seems solid

**Pull request merged**